package.json

```diff
@@ -43,6 +43,7 @@
     "koa-route": "^2.4.2",
     "koa-static": "^1.4.5",
     "lodash": "^4.6.1",
+    "lru-cache": "^4.0.1",
```

Comment:
Since this data never expires, how about we simply store it in a variable?  
Saving introducing a new package.